src/modules/config.js

```diff
@@ -362,16 +364,51 @@ const Config = () => {
               },
             },
             // convertTokenValues
+            //{
+            //  name: "convertTokenValues",
+            //  label: "Convert Tokens to $ Value",
+            //  type: "toggle",
+            //  value: cfg.convertTokenValues,
+            //  group: "site-options",
+            //  help: {
+            //    label: "?",
+            //   text: `<p>Enabling this option will show all token values as $ values</p>`,
+            //  },
+            //},
+            // enableUpdateChecks & updateCheckFrequency
             {
               name: "convertTokenValues",
-              label: "Convert Tokens to $ Value",
+              label: "Convert Tokens to $",
               type: "toggle",
               value: cfg.convertTokenValues,
               group: "site-options",
               help: {
                 label: "?",
-                text: `<p>Enabling this option will show all token values as $ values</p>`,
+                text: `<p>Enabling this option will show all token values as $</p>`,
               },
+              config: {
+                title: "Exchange Rate",
+                options: [
+                  {
+                    type: "number",
+                    valid: "number",
+                    label: "Rate",
+                    name: "usdExchangeRate",
+                    help: {
+                      label: "?",
+                      title: "Currency Exchange Rate",
+                      text: `<p>Whats your exchange rate to USD?</p>
```

Comment:
Might be worth using this API to automatically convert the values
https://freecurrencyapi.com/